bin/as-recon-discover-services.tengo

```diff
@@ -0,0 +1,195 @@
+#! /usr/bin/env arsenic
+
+// arsenic := import("arsenic")
+cobra := import("cobra")
+os := import("os")
+os2 := import("os2")
```

Comment:
why os2?

Reply:
`os` is a module that tengo created. They only implemented some parts of the `os` golang package so `os2` contains the other parts of the `os` golang package that I wanted to use plus some other custom things. We can name it something different, I just didn't know what to call it haha.